src/core/geometry/qgsgeometryutils.h

```diff
@@ -291,6 +291,31 @@ class CORE_EXPORT QgsGeometryUtils
      */
     static QgsPointV2 midpoint( const QgsPointV2& pt1, const QgsPointV2& pt2 );
 
+    /** Return the slope of a line defined by points pt1 and pt2.
+     * @param pt1 first point.
+     * @param pt2 second point.
+     * @return The slope of this linear entity, or infinity if vertical
+     * @note added in QGIS 3.0
+     */
+    static double slope( const QgsPointV2& pt1, const QgsPointV2& pt2 );
+
+    /** Return the coefficients (a, b, c for equation "ax + by + c = 0") of a line defined by points pt1 and pt2.
+     * @param pt1 first point.
+     * @param pt2 second point.
+     * @return The coefficients of this linear entity
+     * @note added in QGIS 3.0
+     */
+    static QVector<double> coefficients( const QgsPointV2& pt1, const QgsPointV2& pt2 );
```

Comment:
OK. Do you want the same correction for:
QVector<double> lengths() const;
QVector<double> angles() const;
QVector<QgsLineString *> altitudes( ) const;
QVector<QgsLineString *> medians( ) const; 
QVector<QgsLineString *> bisectors( double lengthTolerance = 0.0001 ) const;